client/components/SignIn.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {Route} from 'react-router-dom'
+
+class SignIn extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      username: '',
+      password: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+  }
+  handleChange(e) {
+    this.setState({
+      username: e.target.value,
+      password: e.target.value
+    })
+  }
+}
+
+// signedIn (e) {
+//   e.preventDefault()
+//   this.userIn(this.state.username),
+//   this.pass(this.state.password)
+// }
+
+render() {
+  return (
+  <div className='signIn'>
+    <div className='signIn-form'>
+    <form>
+      <label>
+        username:
+        <input type="text" name="username" />
+      </label>
+        <label>signedIn
+        Password:
+        <input type="password" name="password" />
+      </label>
+        <input onChange={this.handleChange(e)} type="submit" value="Submit" />
```

Comment:
Should be `onChange={this.handleChange}`

Also, change `value="Submit"` to `value="Sign in"`